db/migrations/20200504131504_update_all_urns_function.sql

```diff
@@ -0,0 +1,80 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+
+DROP FUNCTION api.all_urns;
+
+CREATE OR REPLACE FUNCTION api.all_urns(block_height bigint DEFAULT api.max_block(), max_results integer DEFAULT NULL::integer, result_offset integer DEFAULT 0) RETURNS SETOF api.urn_snapshot
+    LANGUAGE sql STABLE
+    AS $$
+SELECT urn_identifier, ilk_identifier, all_urns.block_height, ink, coalesce(art, 0), created, updated
```

Comment:
I think we need this to be distinct on the urn_identifier + ilk_identifier so that we're only getting the most recent state of every urn in the system

Reply:
Does his mean you can have the same urn_identifier with a different ilk, and that should be a different urn?

Reply:
👍